packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -301,7 +300,8 @@ export default class Tooltip extends PureComponent<TooltipProps> {
       tip,
       arrowClassName,
       id,
-      "data-testid": dataTestId
+      "data-testid": dataTestId,
+      position
```

Comment:
why was it added?

Reply:
In order to send position to Dialog as DialogPosition